src/main/routes/routes.ts

```diff
@@ -2,13 +2,24 @@ import os from 'os';
 
 import { infoRequestHandler } from '@hmcts/info-provider';
 import { Application } from 'express';
+import { FileFilterCallback } from 'multer';
 
+import { AppRequest } from '../definitions/appRequest';
 import { InterceptPaths, PageUrls, Urls } from '../definitions/constants';
 
 const multer = require('multer');
 const handleUploads = multer({
   limits: {
-    fileSize: 40000000,
+    fileSize: 314572800,
```

Comment:
Magic Number, could refactor this into a const with a good name (like LIMIT_300MB). This is referenced in two places